src/styles/TrainStats.css

```diff
@@ -0,0 +1,34 @@
+.stats-container {
```

Comment:
Great point about scope. Thanks for raising that.

(1) I also don't have strong feelings. I built that way because it is the structure used on my previous job. Looking at https://reactjs.org/docs/faq-structure.html we can go for the Atomic Design/repos you mentioned.

(2) I like option 2, but it seems a bit more work maybe for its own ticket. I will give option 3 a try